examples/platform/k32w/app/support/Memconfig.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#ifndef SRC_PLATFORM_K32W_MEMCONFIG_H_
+#define SRC_PLATFORM_K32W_MEMCONFIG_H_
+
+extern "C" {
+size_t xPortMallocUsableSize(void * pv);
+void * __wrap_malloc(size_t size);
+void __wrap_free(void * ptr);
+void * __wrap_calloc(size_t num, size_t size);
+void * __wrap_realloc(void * ptr, size_t new_size);
+void * __wrap__malloc_r(void * REENT, size_t size);
```

Comment:
Some of these are even spelled incorrectly (`__wrap__malloc_r` instead of `__wrap_malloc_r`)